libs/stocks/feature-shell/src/lib/stocks/stocks.component.html

```diff
@@ -27,6 +27,15 @@
         {{ timePeriod.viewValue }}
       </mat-option>
     </mat-select>
+    <mat-error
+      ><span
+        *ngIf="
+          !stockPickerForm.get('period').valid &&
+          stockPickerForm.get('period').touched
+        "
+        >Please select period</span
+      >
+    </mat-error>
```

Comment:
period element is also required field in form definition but inline error is not handled for the same